index.js

```diff
@@ -3,32 +3,257 @@ const dropDown = document.getElementById('drop-down');
-  } else {
+if (window.innerWidth < 768) {
+  const toggleDropDown = () => {
+    isShown = !isShown;
+    if (isShown === true) {
+      dropDown.style.transform = 'translateX(0%)';
+      menuBar.childNodes[1].style.color = 'white';
+      menuBar.childNodes[1].className = 'fa-regular fa-x';
+      menuBar.childNodes[1].style.fontSize = '20px';
+      document.body.style.position = 'fixed';
+    } else {
+      dropDown.style.transform = 'translateX(100%)';
+      menuBar.childNodes[1].style.color = '#6070ff';
+      menuBar.childNodes[1].className = 'fa-solid fa-bars';
+      menuBar.childNodes[1].style.fontSize = '30px';
+      document.body.style.position = '';
+    }
+  };
+  menuBar.addEventListener('click', toggleDropDown);
+  const removeModal = () => {
     dropDown.style.transform = 'translateX(100%)';
     menuBar.childNodes[1].style.color = '#6070ff';
     menuBar.childNodes[1].className = 'fa-solid fa-bars';
     menuBar.childNodes[1].style.fontSize = '30px';
-  }
-};
-menuBar.addEventListener('click', toggleDropDown);
-const removeModal = () => {
-  dropDown.style.transform = 'translateX(100%)';
-  menuBar.childNodes[1].style.color = '#6070ff';
-  menuBar.childNodes[1].className = 'fa-solid fa-bars';
-  menuBar.childNodes[1].style.fontSize = '30px';
-  isShown = false;
+    isShown = false;
+    document.body.style.position = '';
+  };
+
+  portfolioLink.addEventListener('click', removeModal);
+  aboutLink.addEventListener('click', removeModal);
+  contactLink.addEventListener('click', removeModal);
+}
+// Creating code for dynamic rendering of projects
+
+const projectLists = [
+  {
+    id: '1',
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    imageLink: './assets/first-image.jpg',
+    alt: 'first project image',
+    tech: ['html', 'css', 'javascript'],
+    liveVersion: '#',
+    source: '#',
+  },
+  {
+    id: '2',
+    name: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    imageLink: './assets/second-image.png',
+    alt: 'second project image',
+    tech: ['html', 'css', 'javascript'],
+    liveVersion: '#',
+    source: '#',
+  },
+  {
+    id: '3',
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    imageLink: './assets/third-image.jpg',
+    alt: 'third project image',
+    tech: ['html', 'css', 'javascript'],
+    liveVersion: '#',
+    source: '#',
+  },
+  {
+    id: '4',
+    name: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    imageLink: './assets/fouth-image.jpg',
+    alt: 'fourth project image',
+    tech: ['html', 'css', 'javascript'],
+    liveVersion: '#',
+    source: '#',
+  },
+];
+
+const togglePopUp = (e) => {
+  const itemId = e.originalTarget.id;
```

Comment:
- [ ] It is true that the popup is not displayed when the button is clicked. This is because you are using a property that is not compatible with other browsers but only Mozilla. Please read the content of the screenshot below:

  ![image](https://user-images.githubusercontent.com/45002654/195369141-7c810ab1-670c-435f-8514-1e69348eacf1.png)
  
  Please be advised against using the latest browser properties that have not been made a standard. If this was a business website, then you would lose a lot of clients because of this one innocent mistake.
  
  Consider refactoring this part of code as so 👇🏼:
  
```suggestion
  const itemId = e.target.id;
```

Reply:
ok @Lameck1 thanks a lot. i'll be sure to apply the tip.